### PR TITLE
88 update all text and icon colors to new values

### PR DESCRIPTION
**Pull request by Gowrisankar1**
I have updated all text and icon colors to new values

### CODE REVIEW

**Review by ajayesivan — CHANGES_REQUESTED** [review]
For this task, let's leverage the power of CSS variables. I suggest defining two color variables, text-primary-color and text-secondary-color, and utilizing them throughout the code instead of hardcoding the values. This approach will enhance code maintainability and flexibility.

Reference: https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties

**Comment by Gowrisankar1** [comment]
I have done the changes. Please take a look.

File: public/styles.css [hunk 1]
```diff
@@ -47,7 +48,7 @@
   background-color: #ffffff;
   border-radius: 10px;
   display: block;
-  color: #000000;
+  color: #d81010;
```
Comment (ajayesivan) [inline public/styles.css:51]: @Gowrisankar1 Could you please explain this change?

File: public/styles.css [hunk 2]
```diff
@@ -739,4 +744,19 @@ html {
   .home {
     height: 78vh;
   }
+}
+.secondary{
+
+color: var(--text-seocndary-color);
+}
+.primary{
```
Comment (ajayesivan) [inline public/styles.css:752]: Please format your code. You can use[ VScode's inbuild format feature](https://code.visualstudio.com/docs/editor/codebasics#_formatting).

File: public/terms-and-conditions.html [hunk 3]
```diff
@@ -115,20 +115,21 @@ <h6>6. Limitation of Liability</h6>
     </section>
     <section class="download-section">
       <div class="download-content">
-        <p>Download now and start effortlessly<br>tracking your expenses.</p>
+        <p>Download now and start effortlessly tracking your expenses.</p>
```
Comment (ajayesivan) [inline public/terms-and-conditions.html:118]: Why does this PR have unrelated changes?

**Review by ajayesivan — CHANGES_REQUESTED** [review]

**Comment by ajayesivan** [comment]
@Gowrisankar1 avoid putting the issue number in the PR title.

File: public/styles.css (hunk 1 above)
Comment (ajayesivan) [reply public/styles.css:51]: Not sure what you meant by the screenshot 🤔

Comment (Gowrisankar1) [reply public/styles.css:51]: I didn't changed it I was just checking

Comment (Gowrisankar1) [reply public/styles.css]: I changed it.Please take a look

**Review by ajayesivan — APPROVED** [review]
LGTM! Thanks for the PR @Gowrisankar1

Please resolve the conflicts and merge the PR.

**Comment by Gowrisankar1** [comment]
Can i squash and merge the PR

**Comment by ajayesivan** [comment]
> Can i squash and merge the PR

Yep.

**Pull request merged**